docs/tern.shapes.ttl

```diff
@@ -1067,7 +1067,6 @@ tern-shapes:Result-isResultOf
     sh:description "Relation linking a Result to the Observation or Actuation or act of Sampling that created or caused it." ;
     sh:maxCount 1 ;
```

Comment:
@dr-shorthair can you think of any scenarios where a result can be from multiple activities?

Reply:
The restriction from SOSA suggests it may be a result from more than one activity. 
https://www.w3.org/TR/vocab-ssn/#SOSAResult
> [sosa:isResultOf](https://www.w3.org/TR/vocab-ssn/#SOSAisResultOf) MIN 1

Reply:
An obvious case would relate to 'sub-activities' - i.e. where a larger activity is divided into smaller components.